packages/pieces/community/legis/src/lib/actions/create-ticket.ts

```diff
@@ -320,40 +322,47 @@ export const createTicket = createAction({
       {
         params: {
           where: {
-            ticketTypeFormResponseId: numberAdditionalProcess.data[0].id
+            ticketTypeFormResponseId: numberAdditionalProcess.data[0].id,
+            ticketTypeId: 100047
```

Comment:
Definir essa constante lá no comecinho do código algo como:

const TICKET_TYPE_ID_-nome_definido- = 100047


```suggestion
            ticketTypeId: TICKET_TYPE_ID_-nome_definido-
```